### PR TITLE
ZENKO-1121 feat: reassign topic partitions

### PR DESCRIPTION
**Pull request by giacope**
**What does this PR do, and why do we need it?**

Updates the Zenko-Queue chart to add a new feature to the topic configuration values: `reassignPartitions`. When this value is set to `true`, the `zenko-queue-config` `job` will reassign partitions and increase the replication factor of an existing topic.

**Which issue does this PR fix?**

fixes ZENKO-1121

**Special notes for your reviewers**:

### CODE REVIEW

**Review by tmacro — APPROVED** [review]

File: kubernetes/zenko/charts/zenko-queue/values.yaml [hunk 1]
```diff
@@ -308,6 +308,8 @@ prometheus:
 ## Topic creation and configuration.
 ## The job will be run on a deployment only when the config has been changed.
 ## - If 'partitions' and 'replicationFactor' are specified we create the topic (with --if-not-exists.)
+## - If 'partitions', 'replicationFactor' and 'reassignPartitions' are specified we reassign the partitions to
+## increase the replication factor of an existing topic.
```
Comment (rahulreddy) [inline kubernetes/zenko/charts/zenko-queue/values.yaml:312]: An extra note would help saying, if increasing replication factor, please set reassignPartitions: true.

**Review by rahulreddy — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e wait

File: kubernetes/zenko/charts/zenko-queue/values.yaml (hunk 1 above)
Comment (giacope) [reply kubernetes/zenko/charts/zenko-queue/values.yaml:312]: Added a note below the section to don't break the format.

File: kubernetes/zenko/charts/zenko-queue/templates/configmap-config.yaml [hunk 2]
```diff
@@ -21,7 +21,19 @@ data:
     done
     echo "Applying runtime configuration using {{ .Values.image }}:{{ .Values.imageTag }}"
     {{- range $n, $topic := .Values.topics }}
-    {{- if and $topic.partitions $topic.replicationFactor }}
+    {{- if and (and $topic.reassignPartitions $topic.replicationFactor) $topic.reassignPartitions }}
```
Comment (jonathan-gramain) [inline kubernetes/zenko/charts/zenko-queue/templates/configmap-config.yaml]: is the extra `and` needed? As `$topic.reassignPartitions` is duplicated.

Comment (giacope) [reply kubernetes/zenko/charts/zenko-queue/templates/configmap-config.yaml]: oh, good catch. It's supposed to be $topic.partitions. That's why the extra `and`

Comment (giacope) [reply kubernetes/zenko/charts/zenko-queue/templates/configmap-config.yaml]: However, I just tested and `and` works with a multiple number of arguments.

**Review by jonathan-gramain — APPROVED** [review]

**Comment by giacope** [comment]
Closing this because #320 has the commit for it, and it will reduce CI wait time.

**Pull request closed without merging**